G/Ghostscript/build_tarballs.jl

```diff
@@ -49,6 +47,9 @@ make -j${nproc}
 # install to prefixes
 make install
 install -Dvm 755 "sobin/libgs.${dlext}" "${libdir}/libgs.${dlext}"
+install -Dvm 755 "psi/iapi.h" "${includedir}/ghostscript/iapi.h"
+install -Dvm 755 "psi/ierrors.h" "${includedir}/ghostscript/ierrors.h"
+install -Dvm 755 "base/gserrors.h" "${includedir}/ghostscript/gserrors.h"
```

Comment:
These files shouldn't be executable
```suggestion
install -Dvm 644 "psi/iapi.h" "${includedir}/ghostscript/iapi.h"
install -Dvm 644 "psi/ierrors.h" "${includedir}/ghostscript/ierrors.h"
install -Dvm 644 "base/gserrors.h" "${includedir}/ghostscript/gserrors.h"
```
Honestly it's quite weird we have to do manually install this stuff in the first place.  What do Linux distributions do here?

Reply:
Yes, good catch! There is an `soinstall` step that can just take care of this